docker/docker-entrypoint.sh

```diff
@@ -13,10 +13,11 @@ fi
 
 # Add `java -jar /wiremock-standalone.jar` as command if needed
 if [ "${1:0:1}" = "-" ]; then
-	set -- java -Dlog4j.configurationFile="/var/wiremock/lib/log4j2.xml" \
+	set -- java -Xms2048m -Xmx3072m -Dlog4j.configurationFile="/var/wiremock/lib/log4j2.xml" \
```

Comment:
this requires us to also adapt the k8s settings. see: https://github.com/deliveryhero/rps-system-engineering/blob/master/kubernetes/charts/apps/courier-service-wiremock/values.yaml